### PR TITLE
add Belarus translations

### PR DESCRIPTION
**Pull request by Sadovnikov94**
- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
Some tests failed.
See the logs https://travis-ci.org/jhipster/generator-jhipster/jobs/404156435

Can you have a look plz ?

**Comment by gmarziou** [comment]
I think most Travis errors come from lack of translations for entities, they should be in generators/entity-i18n/templates/i18n

**Comment by Sadovnikov94** [comment]
@gmarziou thanks, you are right. Adding this file have fixed the build

File: generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+{
+    "global": {
+        "title": "<%= capitalizedBaseName %>",
+        "browsehappy": "Dаш браўзэр <strong>састарэў</strong>. Калі ласка <a href=\"http://browsehappy
+        .com/?locale=ru\">абновіце ваш браўзэр</a>.",
```
Comment (pascalgrimaud) [inline generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs]: it should not be a carriage return here

File: generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            "email": "Email",
+            "email.placeholder": "Ваш email"
+        },
+        "messages": {
+            "info": {
+                "authenticated": {
+                    "prefix": "Калі вы хочаце ",
+                    "link": "аўтарызавацца",
+                    "suffix": ", вы можаце паспрабаваць акаўнты па змаўчанні:<br/>- Адміністратар (лагін=\"admin\" і
+                    пароль=\"admin\") <br/>- Карыстальнік (лагін=\"user\" і пароль=\"user\")."
```
Comment (pascalgrimaud) [inline generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs]: same comment: it should not be a carriage return here

File: generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs (hunk 1 above)
Comment (Sadovnikov94) [reply generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs]: Done

File: generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs (hunk 2 above)
Comment (Sadovnikov94) [reply generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs]: Done

**Comment by pascalgrimaud** [comment]
Thanks @Sadovnikov94 and congrats for your 1st contribution !

**Pull request merged**